std/plugins/dispatch_json/mod.ts

```diff
@@ -0,0 +1,103 @@
+import { assert } from "../../testing/asserts.ts";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Ok = any;
+
+interface JsonError {
+  message: string;
+}
+
+interface JsonResponse {
+  ok?: Ok;
+  err?: JsonError;
+  promiseId?: number; // Only present in async messages.
+}
+
+export interface ResolvableMethods<T> {
```

Comment:
Deferred? https://github.com/denoland/deno/blob/31ddfd5a42d658b92e954e44d3326a8e37ac9198/std/util/async.ts#L8

Reply:
Didn't know this already existed in std under a different name. 👍